index.html

```diff
@@ -55,12 +55,65 @@
     </select>
   </div>
   </main>
-  <footer class="page-footer footer-light transparent">
+  <footer class="page-footer">
     <div class="row center for-mobile"> <a class="github-button" href="{{ site.githuburl }}" data-size="large">Fork</a> </div>
-    <div class="row">
-      <div class="col m6 offset-m3 center"> <img class="footer-logo" src="https://api.coala.io/en/latest/_static/images/coala_logo.svg" alt="coala"> <span class="footer-title">{{ site.organization }}</span> </div>
+    <div class="container">
+        <div class="row">
+            <div class="col m3 s12">
+                <h5 class="white-text fine">Site Links</h5>
+                <ul>
+                    <li><a class="grey-text text-lighten-3" href="http://coala.io/docs">Documentation</a></li>
+                    <li><a class="grey-text text-lighten-3" href="http://coala.io/newcomer">Newcomer Docs</a></li>
+                    <li><a class="grey-text text-lighten-3" href="http://coala.io/coc">Code of Conduct</a></li>
+                    <li><a class="grey-text text-lighten-3" href="http://blog.coala.io/">Blog</a></li>
+                </ul>
+            </div>
+            <div class="col m9 s12 center">
+              <div class="center">
+                <h3 class="fine white-text">
+                  <span>
+                    <img class="footer-logo" src="https://api.coala.io/en/latest/_static/images/coala_logo.svg" alt="coala">
+                  </span>
+                  <span>{{ site.organization }}</span>
+                </h3>
+              </div>
+              <div class="center footer-icons">
+                <span class="icon">
+                  <a href="https://twitter.com/coala_io" target="_blank" class="btn center">
+                    <i class="fa fa-twitter" aria-hidden="true">
+                      Follow us on Twitter
+                    </i>
+                  </a>
+                </span>
+                <span class="icon">
+                  <a href="https://gitter.im/coala/coala/gsoc" target="_blank" class="btn center">
+                    <i class="fa fa-commenting-o" aria-hidden="true">
+                      Join Chat on Gitter
+                    </i>
+                  </a>
+                </span>
+                <span class="icon">
+                  <a href="https://github.com/coala/coala/" target="_blank" class="btn center">
+                    <i class="fa fa-github" aria-hidden="true">
+                      Star us on GitHub
+                    </i>
+                  </a>
+                </span>
+              </div>
+            </div>
+          </div>
+        </div>
+    </div>
+    <div class="footer-copyright">
+        <div class="container">
+            <span class="left">
+                ©
+                <span id="current-year"></span>
+                coala
+            </span>
+            <span class="grey-text text-lighten-4 right">GNU AGPL v3.0</span>
```

Comment:
Why not using `{{ site.license }}` ?  If you want a shorter text, change the variable value?  Or create a new variable.